Accepted-RFCs/RFC-013-new-initiative-process.md

```diff
@@ -0,0 +1,124 @@
+* We identify some utility we can offer our userbase that either directly or reasonably indirectly builds off of our core mission: to delivery high-quality template resources to people who need help making their own documentation.
+
+Along with that, each new initiative needs someone to lead it. That person needs to be confident in leading (or co-leading) that initiative for at least a year. This is to prevent initiatives from being spun up and left to dwindle. (Unless said initiative isn't intended to last that long, of course. But we can expect some initiatives that intend to be short to take longer than initially planned.)
+
+Finally, each initiative needs to articulate the deliverables it intends to produce in its first year. This can of course mutate over time, but an initial understanding is key to helping the rest of the project understand the new initiative.
+
+With all of that, the initiative should be drafted as a RFC so the project steering committee can both ask insightful questions/provide comments, as flag their possible interest in helping with that initiative.
+
+### Not every doc initiative is a Good Docs Project initiative
+
+As passionate documentarians, we often want to find ways of helping the wider world. That's why we're here! But not every doc project is the right fit to take up project leadership bandwidth and meeting time. So when we look at a new initiative, we should as a group ask:
+
+> Does this initiative naturally fit with our core project? Or is it a "sister" initiative?
+
+For those that might be "sister" initiatives, we want to support those! And part of that support is being honest about if that initiative might be poorly supported as part of our project.
+
+### The timing might not always be right for a given initiative
+
+Keeping to the idea of focusing our efforts and providing deliverables to our userbase, sometimes we'll have a great idea that fits with the Good Docs Project mission that we aren't actually ready for. That could be because we need to process an intensive effort for a release, or we don't have the resources to sustain addressing a cause. Thus, part of this process is about being genuine and honest about what an initiative needs, and if we can provide for that initiative what it needs.
+
+For example, the "Blog/Video" group has dropped videos from their working group for the time being, because the bandwidth to create videos doesn't exist. We may pick that up in 2023 or 2024, or decide the cost of supporting that is better spent on other initiatives.
+
+## Consequences
+
+This could appear to make the project feel "bureaucratic" or "stuffy." While I feel that isn't the case, I understand some ways of implementing this idea could be alienating to passionate people. That's a conversation worth having.
```

Comment:
We're growing as a project, and therefore we need some guidelines to bundle our efforts instead of dispersing into chaos.
I've seen that happening a few times :D  
I greatly appreciate the RFC!